src/frontend/src/tests/lib/utils/nav.utils.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { AppPath, ROUTE_ID_GROUP_APP } from '$lib/constants/routes.constants';
```

Comment:
This shouldn't be part of another PR?

Reply:
Ah yes sure:

- PR #3358

Reply:
Not just the test, you mean the utils as well?

Reply:
correct, this PR is just for the `Hero` logic